### DISCUSSION TITLE
Link to the survey?

### DISCUSSION THREAD

**Discussion by slalaurette** [martinrotter/rssguard#1075] [Q&A] [answered]
I installed the latest devbuild, was prompted with a dialog asking me to fill the survey, clicked on "OK" before realizing I should have clicked on "Go to the survey" instead, and now I can't find a link to it anywhere.

**Comment by martinrotter** [comment, marked as answer]
https://docs.google.com/forms/d/e/1FAIpQLScQ_r_EwM6qojPsIMQHGdnSktU-WGHgporN69mpU-Tvq8y7XQ/viewform
